Application components created

leovaldez08 opened the pull request:
Created necessary HomePage components like NavBar, HeroSection, Carousel and Footer.

leovaldez08 committed:
Application components created

Created necessary HomePage components like NavBar, HeroSection, Carousel and Footer

leovaldez08 committed:
Delete tailwind.config.js

removed tailwind config file

yash0212 commented on FrontEnd/index.html:
```diff
@@ -2,9 +2,15 @@
+    />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Vite + React</title>
+    <title>PromptVers</title>
```
Is there a typo here?

leovaldez08 replied:
Oops, I'll fix it in the next PR, if that's ok!

yash0212 reviewed (commented):
Please respond to the comment and add a fix if required

leovaldez08 committed:
Update index.html

leovaldez08 committed:
Updated NavBar and HeroSection

Finished static part of the UI components i.e, NavBar and HeroSection. Added favicon for the website. Also added necessary SVGs to the assets folder.

leovaldez08 committed:
Revert "Updated NavBar and HeroSection"

This reverts commit 54b3c7d7877b6591f9c25cbd9d5da3115d38d487.

leovaldez08 committed:
Added Favicons

Added Favicon tags using SVGs

jithin-kalvian reviewed (approved):
LGTM
